tests/integration/modules/grains.py

```diff
@@ -111,9 +111,14 @@ def test_get_core_grains(self):
         test to ensure some core grains are returned
         '''
         grains = ['os', 'os_family', 'osmajorrelease', 'osrelease', 'osfullname', 'id']
+        os = self.run_function('grains.get', ['os'])
+
         for grain in grains:
             get_grain = self.run_function('grains.get', [grain])
-            self.assertTrue(get_grain, grain + "is not available")
+            if os == 'Arch' and grain in ['osmajorrelease', 'osrelease']:
+                self.assertEqual(get_grain, '')
+                continue
+            self.assertTrue(get_grain)
```

Comment:
Because when the grain doesn't exist it doesn't return that message. it just returns empty. 

```bash
salt-call --local grains.item osmajorrelease
local:
    ----------
    osmajorrelease:
```
```bash
salt --versions-report
Salt Version:
           Salt: 2016.3.1

Dependency Versions:
           cffi: 1.7.0
       cherrypy: 3.8.0
       dateutil: 2.4.2
          gitdb: 0.6.4
      gitpython: 1.0.1
          ioflo: 1.5.0
         Jinja2: 2.8
        libgit2: Not Installed
        libnacl: 1.4.4
       M2Crypto: 0.24.0
           Mako: Not Installed
   msgpack-pure: Not Installed
 msgpack-python: 0.4.7
   mysql-python: 1.2.5
      pycparser: 2.14
       pycrypto: 2.6.1
         pygit2: Not Installed
         Python: 2.7.11 (default, Mar 31 2016, 06:18:34)
   python-gnupg: 0.3.7
         PyYAML: 3.11
          PyZMQ: 15.2.0
           RAET: 0.6.5
          smmap: 0.9.0
        timelib: 0.2.4
        Tornado: 4.3
            ZMQ: 4.1.4

System Versions:
           dist:
        machine: x86_64
        release: 4.1.3
         system: Linux
        version: Not Installed
```